docs/docs/build/getting-started/helloworld.mdx

```diff
@@ -151,7 +486,11 @@ To deploy the compiled coin smart contract first open the MetaMask extension and
 
 Choose the Remix `ENVIRONMENT` and `Injected Provider - MetaMask`. Remix will deploy contract to connected network, i.e., BOB.
 
-Select contract as `Coin` click `Deploy` and sign the transaction pop up message on .
+Select contract as `HelloBitcoin`
+
+Add the `_RELAY` ie BitcoinRelay and `_USDTCONTRACTADDRESS` address as parameter.
```

Comment:
What are the addresses that I should add there?

Reply:
added `TestLightRelay` contract for remix